db/migrate/20240611091030_create_all_events.rb

```diff
@@ -0,0 +1,5 @@
+class CreateAllEvents < ActiveRecord::Migration[7.1]
+  def change
+    create_view :all_events
+  end
+end
```

Comment:
i thought that scenic would allow us to keep a different dir/file structure for migrations? Joel, on behalf of imtiaz via tuple

Reply:
so you get a different directory for the SQL definitions, but it still needs to create a migration file to apply, this allow them to leverage the existing versioning stuff... not 100% sure I agree, but also not terrible..